inc/core/patterns/bcgov-alternating-cards.php

```diff
@@ -12,7 +12,8 @@
 return array(
```

Comment:
We prefer [] over `array()` syntax for PHP. The linter should be looking for it though. So not a blocker if it wasn't auto-flagged